test/e2e/lifecycle/cluster_upgrade.go

```diff
@@ -52,13 +52,13 @@ var upgradeTests = []upgrades.Test{
 	&upgrades.AppArmorUpgradeTest{},
 }
 
-var _ = framework.KubeDescribe("Upgrade [Feature:Upgrade]", func() {
+var _ = SIGDescribe("Upgrade [Feature:Upgrade]", func() {
 	f := framework.NewDefaultFramework("cluster-upgrade")
 
 	// Create the frameworks here because we can only create them
 	// in a "Describe".
 	testFrameworks := createUpgradeFrameworks()
-	framework.KubeDescribe("master upgrade", func() {
+	SIGDescribe("master upgrade", func() {
 		It("should maintain a functioning cluster [Feature:MasterUpgrade]", func() {
 			upgCtx, err := getUpgradeContext(f.ClientSet.Discovery(), framework.TestContext.UpgradeTarget)
 			framework.ExpectNoError(err)
```

Comment:
3 lines below this that I can't comment on (because github), you might want to change it to

```go
masterUpgradeTest := &junit.TestCase{Name: "[sig-cluster-lifecycle] master-upgrade", Classname: "upgrade_tests"}
```

It happens elsewhere in this file too. Just don't change it for the individual upgrade tests.